mobile/ios/Podfile.lock

```diff
@@ -18,6 +18,7 @@ PODS:
   - AppCenterReactNativeShared (4.4.5):
     - AppCenter/Core (= 4.4.3)
   - Base64 (1.1.2)
+  - BEMCheckBox (1.4.1)
```

Comment:
Looks like this PR is from before we fixed the issue with pods changing. Would probably need to merge the latest from dev and run `pod install` again

Reply:
I submitted a new PR with the podfile from the dev branch! Had to copy and paste the podfile from github because merging the latest from dev and running pod install did not work.